FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/BasicClient.java

```diff
@@ -203,6 +208,11 @@ public GenericBuilder setTestModeCustomUrl(String serverCustomUrl) {
             return this;
         }
 
+        public GenericBuilder setTestModeResourceUrl(String resourceCustomUrl) {
```

Comment:
**[MINOR]**
When there was only one URL that can be defined, then 'setTestModeCustomUrl' without specifying what url it is works. Would it make sense to change the method name to 'setTestModeServerCustomUrl', and also for resource service url method?

Reply:
Done.